scripts/who_is_better.coffee

```diff
@@ -0,0 +1,110 @@
+#   hubot кто молодец в прошлом месяце - Покажет список сотрудников с их отработанным временем, за прошлый месяц
+#
+# Author:
+#   artem.telnov@kodep.ru
+#
+
+request = require('axios')
+Promise = require('bluebird')
+
+module.exports = (robot) ->
+  durationService = new QuiryAPI()
+
+
+  # Duration This Month
+  #
+  robot.respond /(кто молодец в этом месяце)/i, (res) ->
+    durationService.getCount('duration', 'current')
+    .then((result) -> res.send(result))
+    .catch((err) ->
+      robot.logger.error err
+      res.reply 'Я занят, попросите позже'
+    )
+
+  robot.router.post '/hubot/duration_this_month', (req, res) ->
+    durationService.getCount('duration', 'current')
+    .then((result) -> res.json({text: result}))
+    .catch((err) ->
+      robot.logger.error err
+      res.status(500).json({text: 'Я занят, попросите позже'})
+    )
+  # Approved This Month
+  #
+  # robot.respond /(кто самый оплачиваемый)/i, (res) ->
+  #   durationService.getCount('approved_duration', 'current')
+  #   .then((result) -> res.send(result))
+  #   .catch((err) ->
+  #     robot.logger.error err
+  #     res.reply 'Я занят, попросите позже'
+  #   )
+  #
+  # robot.router.post '/hubot/approved_duration_this_month', (req, res) ->
+  #   durationService.getCount('approved_duration', 'current')
+  #   .then((result) -> res.json({text: result}))
+  #   .catch((err) ->
+  #     robot.logger.error err
+  #     res.status(500).json({text: 'Я занят, попросите позже'})
+  #   )
+
+  # Duration Previous Month
+  #
+  robot.respond /(кто молодец в прошлом месяце)/i, (res) ->
+    durationService.getCount('duration', 'previous')
+    .then((result) -> res.send(result))
+    .catch((err) ->
+      robot.logger.error err
+      res.reply 'Я занят, попросите позже'
+    )
+
+  robot.router.post '/hubot/duration_previous_month', (req, res) ->
+    durationService.getCount('duration', 'previos')
+    .then((result) -> res.json({text: result}))
+    .catch((err) ->
+      robot.logger.error err
+      res.status(500).json({text: 'Я занят, попросите позже'})
+    )
+
+  # Approved Previous Month
+  #
+  # robot.respond /(кто был самым оплачиваемый)|(кому мы скажем спасибо за прошлый месяц)|(кто в прошлом месяце принес денег)/i, (res) ->
+  #   durationService.getCount('approved_duration', 'previous')
+  #   .then((result) -> res.send(result))
+  #   .catch((err) ->
+  #     robot.logger.error err
+  #     res.reply 'Я занят, попросите позже'
+  #   )
+  #
+  # robot.router.post '/hubot/approved_duration_previous_month', (req, res) ->
+  #   durationService.getCount('approved_duration', 'previos')
+  #   .then((result) -> res.json({text: result}))
+  #   .catch((err) ->
+  #     robot.logger.error err
+  #     res.status(500).json({text: 'Я занят, попросите позже'})
+  #   )
+
+class QuiryAPI
+  TTL: 5
+  API_URL: 'https://timer.kodep.ru/export_to_jarvis'
```

Comment:
env variables.
for example API_URL: "#{process.env.KODER_TIMER_API}/export_to_jarvis"